src/messaging.js

```diff
@@ -1,18 +1,10 @@
-import {lmsAddress} from './config';
-import Primus from './primus';
-
 export default class Messaging {
-  constructor(tweet, componentId) {
-    console.log('Messaging', componentId);
-
+  constructor(tweet, componentId, localMessaging) {
     this.tweet = tweet;
     this.componentId = componentId;
-    this.isConnected = false;
-  }
 
-  _handleOpenConnection() {
-    console.log('Connection is alive and kicking');
-    this.isConnected = true;
+    this.localMessaging = localMessaging;
+    this.localMessaging.receiveMessages((message) => { this._handleMessage(message) });
```

Comment:
@rodrigopavezi Thoughts on moving `receiveMessages` to another function to avoid the dependency as per `Single Responsibility Principle`?